packages/create-svelte/template-additions/.eslintrc.cjs

```diff
@@ -5,7 +5,7 @@ module.exports = {
 	overrides: [{ files: ['*.svelte'], processor: 'svelte3/svelte3' }],
 	parserOptions: {
 		sourceType: 'module',
-		ecmaVersion: 2018
+		ecmaVersion: 2019
```

Comment:
this needs to be changed in the ts version of that config, too

Reply:
ah, didn't realize there were two. updated